parts/27-unlisted.sh

```diff
@@ -92,3 +92,37 @@ query_cpu-tools-month() {
 			avg_cpu_time_seconds DESC
 	EOF
 }
+
+query_memory-tools-month() {
+	handle_help "$@" <<-EOF
+		Tool Performance Tracking: Memory by Month-Year.
+	EOF
+
+	assert_count $# 1 "Missing host name"
+	host="$1"
+
+	read -r -d '' QUERY <<-EOF
+		WITH totalmem as (
+			SELECT 
+				distinct job_id, 
+				metric_value 
+			FROM 
+				job_metric_numeric
+			WHERE 
+				metric_name = 'memtotal'
+		)
+		SELECT 
+			TO_CHAR(job.create_time, 'YYYY-MM') AS date,
```

Comment:
Generally here what I did instead was 
```suggestion
			date_trunc('month', job.create_time) AS date,
```

to retain it as an actual "month".

Reply:
I'm glad to see you all are retaining the same date thresholds I used in Galactic Radio Telescope, to preserve privacy! It's too bad we couldn't work to align our priorities there a bit more.

Reply:
Agreed - this has been integrated into 22-queries and future date filters should now be referencing the `date_trunc` formula.